sklearn/model_selection/tests/test_split.py

```diff
@@ -819,6 +876,26 @@ def train_test_split_mock_pandas():
     X_train_arr, X_test_arr = train_test_split(X_df)
 
 
+def train_test_split_list_input():
+    # Check that when y is a list / list of string labels, it works.
+    X = np.ones(7)
+    y1 = ['1'] * 4 + ['0'] * 3
+    y2 = np.hstack((np.ones(4), np.zeros(3)))
+    y3 = y2.tolist()
+
+    X_train1, X_test1, y_train1, y_test1 = train_test_split(X, y1, stratify=y1,
```

Comment:
Ah... Thx for the catch... I've clarified that the `stratify` param refers to the class labels not groups...